dashboard/app/assets/stylesheets/application.scss

```diff
@@ -1287,6 +1298,16 @@ a.download-video {
     text-transform: uppercase;
   }
 
+  #heading-banner-text {
+    color: $white;
+    margin-top: 40px;
+  }
+
+  #heading-banner-subtext {
+    color: $white;
+    margin-bottom: 115px;
+  }
```

Comment:
I'm not fully understanding why we have both header-banner (in haml) and HeadingBanner (in React). Are these different things?